### PR TITLE
Sync project deleted status to Data Tracker

### PR DESCRIPTION
**Pull request by mddilley**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/18577

This PR updates the Moped <> Data Tracker sync ETL to include passing the value of `is_deleted` in the `moped_project` table into the `projects` (object_201) Knack table.

## Testing
**URL to test:** 

Local

**Steps to test:**
1. Start up the Moped local stack
2. Log into Knack Builder and go to [this test instance of Data Tracker](https://builder.knack.com/atd/test--austin-transportation-data-tracker--29-jul-2024/records/objects/object_201). This is the table that will receive Moped data from the local stack during these steps.
3. Update [this line](https://github.com/cityofaustin/atd-moped/blob/d7fc8b84d8612ca3559cf66fb969eef32fba75b2/moped-etl/data-tracker-sync/process/knack_data.py#L73) in your local code to `field_4875` since the **Is Deleted** field has different field IDs in the test and the production Knack apps.
4. Follow the instructions in the **Set up the environment file for local testing** and **Testing by running the script locally** sections of the readme to test syncing the creation of a new Moped project and an update to the same project update in the local stack. You might need to create more than one project to arrive at a Moped Project ID that is not in the Knack `projects` table yet. [Sorting the Knack table](https://builder.knack.com/atd/test--austin-transportation-data-tracker--29-jul-2024/records/objects/object_201?builder_table_sort=field_3858%7Cdesc) by **Moped Project ID** descending is helpful to see what is synced in the test app. This isn't super straightforward so please let me know if I can help out with testing.
5. You should see **Is Deleted** is `False` in the Knack table. In the local Moped app, delete your project, and you can test running the ETL locally again to see that value updates to `True`.
6. Read through the **Backfilling a new column** section to see how that will be done after this code releases.

**Deployment plan**
- Production image will update after patch or release. Airflow DAG will begin to populate `is_deleted` for new/updated projects from that point on.
- [ ] Use backfill instructions in this PR to backfill only the `is_deleted` column for the projects in the Knack table.

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Product manager checked [DB view dependencies](https://atd-dts.gitbook.io/moped-documentation/product-management/updating-the-arcgis-online-components-database-view)
~- [ ] Product manager added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable~

### CODE REVIEW

**Review by chiaberry — APPROVED** [review]
Tested and ✅ 

I didnt add a TEST_KNACK_SIGNAL_RECORD_ID to the env file and at first I wasnt sure if that was necessary or not, but it worked without it for the this testing.

**Review by johnclary — COMMENTED** [review]
@mddilley i haven't tested your code, but this looks really straightforward and good. nice updates to the docs 🙏 🚢

File: moped-etl/data-tracker-sync/README.md
Comment (frankhereford) [inline moped-etl/data-tracker-sync/README.md:1]: Thank you for this excellent documentation updates 🙏 📖

**Review by frankhereford — APPROVED** [review]
image
# 🚀 

Your code looks fantastic, and everything makes sense. Testing this would have been really something else without your instructions both on the PR and in the README.

🚢🚢🚢

**Comment by mddilley** [comment]
thanks, y'all - merging!

**Pull request merged**